datafusion/optimizer/src/scalar_subquery_to_join.rs

```diff
@@ -21,7 +21,7 @@ use crate::utils::{
 };
 use crate::{utils, OptimizerConfig, OptimizerRule};
 use datafusion_common::{context, plan_err, Column, Result};
-use datafusion_expr::logical_plan::{Aggregate, Filter, JoinType, Projection, Subquery};
+use datafusion_expr::logical_plan::{Filter, JoinType, Limit, Subquery};
 use datafusion_expr::{combine_filters, Expr, LogicalPlan, LogicalPlanBuilder, Operator};
```

Comment:
The idea here is that if the particular subquery isn't supported by the physical operators, it will error out at that time?

Reply:
> The idea here is that if the particular subquery isn't supported by the physical operators, it will error out at that time?

Yes, exactly.